tests/documentation/test_files/DocumentationStyleBear/bad_file3.py.test.correct

```diff
@@ -0,0 +1,28 @@
+def b():
+    """
+    :param t:
+        xyz
+    :return:
+        123
+    """
+
+
+class Doc():
+    """
+    Example Class
+
+    :param t:
+        xyx
+    :return:
+        215
+    """
+
+    def D1():
```

Comment:
Add `self` or `@staticmethod`.
And `D1` is a violation of PEP8.
Please run *all* your test code through python checkers.  The test code should be good code except for the violation you are intentionally triggering.  Otherwise all I see lots of bad code (in 'good' files) and I cant see which bad code is intentional.